doc/standards/codeStructure.md

```diff
@@ -19,41 +20,67 @@ features
       \
         |- api
           \
-            |- user
-              \
-                |- login.ts ===> caller, adapter
-                |- otherCall.ts
-        |- components ==> UI components specific to the feature
+            |- login.ts ===> caller, adapter
+            |- login.test.ts
+            |- otherCall.ts
+            |- otherCall.test.ts
+        |- components ==> feature specific UI components
           \
-            |- Component1.tsx
-            |- Component1.test.tsx
+            |- Component1
+              \
+                |- Component1.tsx
+                |- Component1.test.tsx
+                |- Component1.stories.tsx
             |- ...
         |- pages ==> feature specific pages
-        |- services ==> utils or data tranform functions
-        fixturesTests.ts ==> data for testing (API call responses mocks)
+          \
+            |- Page1
+              \
+                |- Page1.tsx
+                |- Page2.test.tsx
+            |- ...
+        |- helpers ==> utils or data tranform functions
+        |- fixtures
+          \
+            |- fixture1.ts ==> data for testing (for example API call responses mocks)
+            |- ...
     |- register
     |- homepage
     |- search
 
-libs
+libs ===> external modules implementations
   \
     |- analytics
     |- pushNotification
     |- errorReporting
     |- geolocation
     |- ...
 
+theme ==> theme object used in styled-components
+
 ui
-\
- |- components ===> independent components (reusables)
- |- theme
- |- assets
+  \
+    |- animations ==> lottie animations
+    |- components ===> independent and stateless components (reusables)
+      \
+        |- Component1
+          \
+            |- Component1.tsx
+            |- Component1.test.tsx
+            |- Component1.stories.tsx
+    |- hooks
+    |- svg ==> logos and icons
+    |- styleGuide ==> theme constants and typography components
 ```
 
+
 ### Mistakes to avoid
 
 - Do not put component into `ui/components` if it is used only once in the app
+- Put stories and tests next to corresponding page or component in dedicated subdirectory
```

Comment:
La formulation est fort trompeuse. On est dans `mistakes to avoid` et là il y a une consigne qu'il faut effectivement suivre.